src/app/containers/Article/index.jsx

```diff
@@ -1,75 +1,54 @@
-import React, { Fragment } from 'react';
-import { bool, string, shape } from 'prop-types';
-import Helmet from 'react-helmet';
-import HeaderContainer from '../Header';
+import React from 'react';
+import { string, shape } from 'prop-types';
+import { compose } from 'recompose';
```

Comment:
Reading up on the recompose library it seems that the author recommends not using it for new projects. 
https://github.com/acdlite/recompose#a-note-from-the-author-acdlite-oct-25-2018 
https://github.com/acdlite/recompose/issues/756#issuecomment-438674573

Reply:
Hmm so I read that more as using recompose is still fine, it will not receive new features etc but that there is now an alternative.

I mentioned in the top comment about moving to hooks in the near future. I just wonder if it would be worth finishing up this sort of approach, given we need this functionality relatively soon.

And then refactoring again as this is an MVP.

Reply:
but I am going to try an alternative today, and try and estimate the level of work involved.

Reply:
Actually found this to be a really good read when looking into HOC vs Hooks https://medium.com/@albertchu539/higher-order-components-in-a-react-hooks-world-69fe1f0b0791

I think HOC's and something like `recompose` used here still has its place.

The hooks alternative mentioned by the recompose author is best used when your HOC's were just being used to enhance your base component with props or data derived from state and not actually when you want to render an entirely different component as we do here.

Going without something like recompose would reduce the modularity of these error handling and loading HOC's as we would still have to implement the same if/else logic in each container that wishes to use default error and loading states.

Reply:
As its advised not to use recompose, and I doubt maintainance will be high on anyones list, we should pull out this teeny function into simorgh helpers, with credit ofcourse :P https://github.com/acdlite/recompose/blob/master/src/packages/recompose/compose.js